### PR TITLE
Changes v1.2

### PR DESCRIPTION
**Pull request by Nazeeefa**

### CODE REVIEW

File: pages/cheat_sheets/sequencing.md [hunk 1]
```diff
@@ -118,13 +120,11 @@ https://www.forskningsetikk.no/en/guidelines/science-and-technology/guidelines-f
...
 
 #### RDM Services
-- [ELIXIR helpdesk](https://elixir.no/helpdesk)
-  - Provides assistance with:
-    - Data management planning
-    - Storage and computing
-    - Metadata standards
-    - Data deposition to [ELIXIR databases](https://elixir-europe.org/platforms/data/elixir-deposition-databases)
-  - Contact: ELIXIR helpdesk
+[ELIXIR Norway's HelpDesk](https://elixir.no/helpdesk) can assist you with guidance on:
```
Comment (bianchini88) [inline pages/cheat_sheets/sequencing.md:123]: I would say we are providing much more than "guidance".

Comment (Nazeeefa) [reply pages/cheat_sheets/sequencing.md:123]: True. Maybe we remove 'guidance on' or add something like 'guidance, support, and practical/important tips on:'?

Comment (bianchini88) [reply pages/cheat_sheets/sequencing.md:123]: Yes, just remove "guidance on" and it works!

File: pages/cheat_sheets/sequencing.md [hunk 2]
```diff
@@ -118,13 +120,11 @@ https://www.forskningsetikk.no/en/guidelines/science-and-technology/guidelines-f
...
+  - Data management planning
+  - Storage and computing
+  - Metadata standards
+  - Data deposition to [ELIXIR databases](https://elixir-europe.org/platforms/data/elixir-deposition-databases)
```
Comment (korbinib) [inline pages/cheat_sheets/sequencing.md:127]:
```suggestion
  - Data deposition to [ELIXIR databases](https://elixir-europe.org/platforms/data/elixir-deposition-databases)
  - Data analysis for various sequencing methods
```

**Review by korbinib — CHANGES_REQUESTED** [review]

File: pages/cheat_sheets/sequencing.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: page
 type: cheat_sheet
-title: Next generation sequencing data
+title: Sequencing data
```
[the remark this answers is not in the document]
Comment (korbinib) [reply pages/cheat_sheets/sequencing.md:4]: How about doing both?

[the remark this answers is not in the document]
Comment (Nazeeefa) [reply pages/cheat_sheets/sequencing.md:4]: I am hesitant to use unfamiliar wording in title ('Read'). What's common is "High Throughput Sequencing" and "Short-read and Long-read sequencing or Short/Long-read sequencing" (ignoring NGS).

[the remark this answers is not in the document]
Comment (korbinib) [reply pages/cheat_sheets/sequencing.md:4]: guess "Short-read and Long-read sequencing or Short/Long-read sequencing" is a both a little long.
would "High Throughput Sequencing"  fit?

[the remark this answers is not in the document]
Comment (Nazeeefa) [reply pages/cheat_sheets/sequencing.md:4]: I hope so, High-Throughput Screening fits.

**Pull request merged**